index.tsx

```diff
@@ -47,8 +47,19 @@ const settings = definePluginSettings({
         type: OptionType.BOOLEAN,
         description: "Show notification for 'user sent a message'"
     },
+    showMessageBody: {
+        default: false,
+        type: OptionType.BOOLEAN,
+        description: "Include message contents in notification"
+    },
+    charLimit: {
+        default: 100,
+        type: OptionType.NUMBER,
+        description: "Character limit for notifications. Set to 0 for no limit. Default=100"
+    }
 });
 
+
```

Comment:
This newline comes from the auto-formatting from .eslintrc.json from upstream [Vencord](https://github.com/Vendicated/Vencord).